docs/gitbook/0-overview/README.md

```diff
@@ -0,0 +1,60 @@
+CBRAIN has several important capabilities:
+
+* Files may be be stored anywhere: on local servers or on remote servers.
+* Files are moved automatically and upon demand (e.g. by user or a
+  task), without interfering with platform components and without
+  the user having to worry too much about their actual location.
+* Tasks can be launched at any scale: on local servers or on powerful
+  compute clusters. CBRAIN supports clusters running Sun Grid Engine
+  (SGE), MOAB, Torque/PBS, or simple UNIX processes, transparently.
+  Other adapters are easy to implement.
+* Files can be displayed or accessed on the web interface according
+  to their own internal representation. For example a JPEG file can
+  be viewed, or an MRI file can be parsed slice-by-slice on the web
+  interface.
+* Files and tasks are deployed as plugins, so a CBRAIN administrator
+  can write his own or simply import them from other developers.
+
+## Where does it come from?
+
+The [**CBRAIN service**](https://portal.cbrain.mcgill.ca) was originally conceived
+through a [CANARIE](http://www.canarie.ca)
+grant awarded to [Professor Alan C. Evans](http://mcin-cnim.ca/people/alans-cv/), at
+[McGill University](http://www.mcgill.ca). The resulting internal
+code platform was meant
+to provide easy access to complex neuroimaging computational tools, for
+clinicians or neuroscience researchers with limited IT resources. Despite
+the name, the framework is designed to be general, and can accommodate any
+data and task for any application. This GIT repository contains the
+generic core of the framework, _which is not specific to any field of science_.
```

Comment:
Skimming through this as markdown is kind of awkward because of inconsistent line length formatting. You can get more consistent formatting if you pipe the paragraphs through `fmt` (but fmt tends to eat up markdown lists in my experience, so I would only do it on the paragraphs..)